e2e/test/scenarios/dashboard/reproductions/31274-dashcard-actions-overlap.cy.spec.js

```diff
@@ -57,7 +62,7 @@ describe("issue 31274", () => {
         visibleActionsPanel().should("have.length", 1);
 
         cy.log(
-          "Make sure cypress can click, so element is not covered by another",
+          "Make sure cypress can click the element, which means it is not covered by another",
```

Comment:
There's a typo in the existing test: `mestabase` (line 48, but GitHub won't let me comment on that line).

Reply:
fixed